build.gradle

```diff
@@ -39,25 +41,27 @@ task sourcesJar(type: Jar, dependsOn: classes) {
     from sourceSets.main.allSource
 }
 
-task exposedOracleTest(type: Test) {
-    jvmArgs += "-Dexposed.test.dialects=oracle"
+task exposedDialectTestWithDocker(type: Test) {
+    jvmArgs += "-Dexposed.test.dialects=" + dialect
```

Comment:
I guess it's better to "inject" dialect from env/external property. That will help to setup separate TeamCity build configurations for Oracle and SQL Server

Reply:
But I suppose that `gradle  exposedDialectTestWithDocker -Pdialect=oracle` works fine. And `dialect` is declared in `gradle.properties`